lib/N3Parser.js

```diff
@@ -774,7 +774,13 @@ N3Parser.prototype = {
 
   // ### `_error` emits an error message through the callback
   _error: function (message, token) {
-    this._callback(new Error(message + ' on line ' + token.line + '.'));
+    var err = new Error(message + ' on line ' + token.line + '.');
+    err.context = {
+      token: token,
+      line: token.line,
+      previousToken: this._lexer._prevToken,
```

Comment:
Alternatively, the parser might store it. Would avoid accessing a private variable. (Also, a little strange that the lexer exposes `_prevToken` but not `_token`.)